Juanvvc/c172p-detailed
# Bug 400

Fixes #400 

* Use LF line endings for text files
* Formatted `.xml` files with 4 spaces indentation using the `xmllint` tool

I have processed most of the files, except those in:

* `Models/labels/` 
* `Models/Interior/`
* `Models/Immat/`
* `Models/Pedals/`
* `Models/Liveries/`
* `Models/Effects/`
* `Models/Human/`

I'll fix those remaining files during 3.8. `.nas` should use 4 spaces as well. There's not really a formatting tool for this, so just get rid of those tabs while you working on some `.nas` files.

How to test:

1. Do the usual `git pull`

2. Then `git checkout bug-400`

3. And then `git log -w -p`. Note that you need to use `-w`. It should show only minor changes (changed comments and removal and insertion of blank lines)

4. Just check that you don't see any crazy things.
